packages/config/config/devices/0x0085/zws230.json

```diff
@@ -0,0 +1,29 @@
+// Fakro ZWS230
+// Chain actuator - window opener
+{
+	"manufacturer": "Fakro",
+	"manufacturerId": "0x0085",
+	"label": "ZWS230",
+	"description": "Chain actuator 230VAC",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x0003"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x0002"
+		}
```

Comment:
According to 
https://github.com/OpenZWave/open-zwave/blob/e7b1705403fced2c2d2bc8aa92b559f6351f5542/config/manufacturer_specific.xml#L618-L620
there is type 3, id 2 and type 2, id 5. Where's type 2, id 3 taken from?

Reply:
Type 2 ID 3 (0x0085-0x0002-0x0003) is the actual device that I own.

I attached a nodes-dump snippet of that one node.
[fakro-zws230-nodes.json.gz](https://github.com/zwave-js/node-zwave-js/files/6157184/fakro-zws230-nodes.json.gz)